src/record_signal.cc

```diff
@@ -411,6 +411,29 @@ static void handle_desched_event(RecordTask* t) {
    * That may be a kernel bug, but we handle it by just
    * continuing until we we continue past the arm-desched
    * syscall *and* stop seeing signals. */
+
+  const auto untraced_record_only_exit =
+    uintptr_t(RR_PAGE_SYSCALL_UNTRACED_RECORDING_ONLY) + 4;
+  if (t->arch() == aarch64 && t->ip() == remote_code_ptr(untraced_record_only_exit) &&
```

Comment:
Remove the architecture check?

Reply:
Right. I’ll test on my laptop with a different kernel first.